modules/upgrade.adoc

```diff
@@ -21,7 +21,7 @@ All Kubernetes objects and PVs in each {product-title} cluster are backed up as
 
 [NOTE]
 ====
-The upgrade starting time might be later than the scheduled time. It could be sometimes an hour or longer.
+When following a scheduled upgrade policy, there might be a delay of an hour or more before the upgrade process begins, even if it is an immediate upgrade. Additionally, the duration of the upgrade might vary based on your workload configuration and could take longer than expected.
```

Comment:
I had to re-read this a few times before it was clear to me that the important aspect of the message is that even though a user is following a scheduled upgrade policy that scheduled policy may be delayed. 

I wonder if we could clarify this by saying something like:
When following a scheduled upgrade policy, there might be a delay of an hour or more before the upgrade policy begins. This delay can occur even with scheduled upgrade policy that is immediate.

Reply:
When we say "immediate", it is the action taken by the user and not a status. In the sense, when a user launches the upgrade then and there, it would still take over an hour to begin. Does that answer your question?